Embedded Metadata.js

```diff
@@ -1145,12 +1148,12 @@ var testCases = [
 				"title": "Northwestern Can't Quit ASA Over Boycott Because it is Not a Member",
 				"creators": [
 					{
-						"firstName": "Eugene",
-						"lastName": "Kontorovich",
+						"firstName": "",
+						"lastName": "https://www.facebook.com/EugeneKontorovich?ref=hl",
```

Comment:
I don't understand why this changed here. When is this meta tag extracted and why did it change with your improvement above to extract the `<meta name="author">` tags?

BTW shouldn't in that page `article` then be defined as some namespace? Maybe it is just not the best test example here...